tests/ts-app-template/tests/index.html

```diff
@@ -30,9 +30,14 @@
     <script src="/testem.js" integrity="" data-embroider-ignore></script>
     <script src="/@embroider/core/vendor.js"></script>
     <script src="/@embroider/core/test-support.js"></script>
-    <script src="/@embroider/core/test-entrypoint" type="module"></script>
+    <script type="module">import "ember-testing";</script>
+
+    <script type="module">
+      import { start } from './test-helper';
+      import.meta.glob("./**/*.{js,gjs,gts}", { eager: true });
+      start();
+    </script>
 
     {{content-for "body-footer"}}
-    {{content-for "test-body-footer"}}
```

Comment:
Should this be removed? It's still created in https://github.com/embroider-build/embroider/blob/b059849abbc8c9051192693f5b0fa70a77c836e3/packages/compat/src/content-for-config.ts#L20